packages/php-table-backend-utils/src/Connection/Bigquery/QueryExecutor.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Connection\Bigquery;
+
+use Google\Cloud\BigQuery\BigQueryClient;
+use Google\Cloud\BigQuery\Job;
+use Google\Cloud\BigQuery\JobConfigurationInterface;
+use Google\Cloud\BigQuery\QueryJobConfiguration;
+use Google\Cloud\BigQuery\QueryResults;
+use Retry\BackOff\StatelessBackOffPolicy;
+use Retry\BackOff\UniformRandomBackOffPolicy;
+
+class QueryExecutor
```

Comment:
Abych to mohl rozumně otestovat tak nerozšiřuju klienta, ale udělal sem vlastní třídu.